conda/core/subdir_data.py

```diff
@@ -126,39 +124,59 @@ def __init__(self, channel):
         self.channel = channel
         self.url_w_subdir = self.channel.url(with_credentials=False)
         self.url_w_credentials = self.channel.url(with_credentials=True)
-        self.cache_path_base = join(create_cache_dir(),
-                                    splitext(cache_fn_url(self.url_w_credentials))[0])
+        self.cache_dir = create_cache_dir()
         self._loaded = False
 
     def reload(self):
         self._loaded = False
         self.load()
         return self
 
-    @property
-    def cache_path_json(self):
-        return self.cache_path_base + '.json'
-
-    @property
-    def cache_path_pickle(self):
-        return self.cache_path_base + '.q'
-
     def load(self):
-        _internal_state = self._load()
-        if _internal_state.get("repodata_version", 0) > MAX_REPODATA_VERSION:
-            raise CondaUpgradeError(dals("""
-                The current version of conda is too old to read repodata from
-
-                    %s
+        try:
+            # 1. Try pulling lastmodified.json
```

Comment:
Consider adding feature flag to gate new strategy, especially to make it easier to benchmark against the current full repodata.json strategy.